src/main/java/com/endava/web/application/infrastructure/restcontroller/EmployeeController.java

```diff
@@ -0,0 +1,47 @@
+package com.endava.web.application.infrastructure.restcontroller;
+
+import com.endava.web.application.domain.model.Employee;
+import com.endava.web.application.domain.service.EmployeeService;
+import com.endava.web.application.infrastructure.restcontroller.dto.EmployeeDto;
+import lombok.AllArgsConstructor;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/")
+@AllArgsConstructor
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+    private final Converter<EmployeeDto, com.endava.web.application.domain.model.Employee> employeeDtoEmployeeConverter;
```

Comment:
why do you import fully qualified path ?

Reply:
It was an error. Done.